Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:latest
+RUN apt-get update && apt-get install wget -y
+LABEL org.opencontainers.image.author="Susheel Pal"
+RUN mkdir -p ~/Scripts
+WORKDIR /Scripts
+RUN wget https://github.com/VrushankPatel/Maxine-Server/releases/download/v2.0.0/maxine-discovery-linux
+RUN chmod +x maxine-discovery-linux
+EXPOSE 8080
```

Comment:
Also, Let's not expose port 8080 directly,
Use NGINX layer after this and run it, NGINX will run on PORT 80.
Write nginx.conf so that it'll redirect everything from port 80 to port 8080 where our server runs.
So NGINX will be exposed, backend will keep running inside the container and will not be directly accessible outside.

Reply:
ok